.eslintrc.json

```diff
@@ -1,11 +1,17 @@
 {
     "extends": "./Tools/eslint-config-cesium/browser.js",
     "plugins": [
-        "html"
+        "html",
+        "import"
     ],
+
     "rules": {
-        "no-unused-vars": ["error", {"vars": "all", "args": "none"}]
+        "no-unused-vars": ["error", {"vars": "all", "args": "none"}],
+        "key-spacing": ["error", {"beforeColon": true, "afterColon": true}],
```

Comment:
This is definitely not in our style guide, or at least the style guide is wrong if this is what it says.  We don't put a space before the colon.

Reply:
I'm surprised this isn't in or formatting guide.  We have always done space before and after the colon in this repo.  However, we don't use the space before the colon in any of our other projects, so I would be in favor of removing it here too

Reply:
I've noticed this is consistently inconsistent in the current code base.  From a quick spot check, Source is 7:1 in favor of spaces on both sides, Specs is close to 50/50. My selfish preference is space after and not before, because VSCode isn't configurable for spaces on both sides without using extensions.

Reply:
https://github.com/AnalyticalGraphicsInc/cesium/blob/master/Documentation/Contributors/CodingGuide/README.md#options-parameters

I noticed it here, @lilleyse had me `key: value` to `key : value` in one of my PRs, and we seem to do `get : function()` / `set : function()` in our getters and setters. That's the rationale for this rule.

Reply:
Our eclipse and webstorm configurations explicitly have the autoformatter set to space before colon.  But I agree, I like it better without.  Plus, I think most people on our team have moved to VSCode as our preferred editor

Reply:
All of this bikeshedding is exactly why prettier makes the most sense.

I think some of our PR reviews have gotten overly focused on things like whitespace instead of the code.

Reply:
Correction to my previous comment - Specs is 5:1 in favor of spaces on both sides.  Earlier I wasn't filtering down to just JavaScript.  JSON and glTF files are nearly 100% space after and not before, which was throwing off my results.

Reply:
Just to add to the bikeshedding, I also prefer space after and not before.